pkgs/development/libraries/oracle-instantclient/default.nix

```diff
@@ -61,9 +63,7 @@ in stdenv.mkDerivation rec {
     install -Dm644 sdk/demo/* $out/share/${name}/demo
 
     # PECL::oci8 will not build without this
-    if [ \! -e "$out/lib/libclntsh${extLib}" ]; then
-      ln -s $out/lib/libclntsh${extLib}.12.1 $out/lib/libclntsh${extLib}
-    fi
```

Comment:
Uff, the symlink already seems to exist in the darwin zipfile, but not in the linux one, so the build on darwin would now fail.
We indeed need to check for existence, sorry for the confusion. Could you add these back in, plus a comment above that this symlink only exists in zipfiles for some platforms?

Reply:
What about just overwriting it with `ln -sfn`?

Reply:
Works for me too